README.md

```diff
@@ -172,8 +173,14 @@ After the errors plugin has been initialized on a player, a `timeout()` method i
 
 A new timeout may be set by passing a timeout in milliseconds, e.g. `player.errors.timeout(5 * 1000)`.
 
+Setting the timeout to `Infinity` will turn off this check.
+
 If no argument is passed, the current timeout value is returned.
 
+### `backgroundTimeout()`
```

Comment:
Also, thanks for adding the documentation I forgot :)

Reply:
you're welcome :)